content/install.md

```diff
@@ -1,9 +1,18 @@
 # Install the Wazo Platform UC use case
 
+## Requirements
+
+For a small install of about 20 users, less than 20 calls per day:
+
+- RAM: 2 GiB with 1 GiB swap
```

Comment:
Out of the box (without any data loaded) it already use 1,7G.

I don't think we should talk about swap.

Reply:
ok I just copied what we had in the doc previously but we can change it if needed...

Reply:
@sduthil what do you suggest?

Reply:
Agreed with sileht: Wazo uses 1.7G of RAM out of the box. The swap space was meant as to give some extra room, since the full 1.7G is not continuously needed and most of it can be swapped. I think we can rephrase as:
"RAM: 2 GiB of RAM is a tight minimum"

Reply:
ok added this wording.